sdk/playwrighttesting/create-microsoft-playwright-testing/tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I think we should use the shared base tsconfig here [example](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/documentintelligence/ai-document-intelligence-rest/tsconfig.json#L2) unless there's a reason not to

This will allow consistency across the various `@azure/` package and less config for you to manage